Sources/Metaplex/Drivers/Identity/Identity.swift

```diff
@@ -8,11 +8,23 @@
 import Foundation
 import Solana
 
-public protocol IdentityDriver {
+public protocol IdentityDriver: Account {
```

Comment:
This makes a hard requirement on Solana for the interface.

Reply:
The idea is that at some point solana is not required for Metaplex. We might just import PublicKey and Keypair form a different lib